cachi2/interface/cli.py

```diff
@@ -79,25 +79,18 @@ def cachi2(  # noqa: D103; docstring becomes part of --help message
         resolve_path=True,
         readable=True,
     ),
+    log_level: LogLevel = typer.Option(
+        LogLevel.INFO,
```

Comment:
minor detail: in `cachi2 --help`, the default looks like this

```
  --log-level [DEBUG|INFO|WARNING|ERROR|CRITICAL]
                                  Set log level.  [default: LogLevel.INFO]
```
If you change the default to `LogLevel.INFO.value`, it will be
```
  --log-level [DEBUG|INFO|WARNING|ERROR|CRITICAL]
                                  Set log level.  [default: INFO]
```
which is what we want

Reply:
Fixed